impeller/renderer/backend/vulkan/tracked_objects_vk.cc

```diff
@@ -50,6 +50,13 @@ void TrackedObjectsVK::Track(std::shared_ptr<const DeviceBuffer> buffer) {
   tracked_buffers_.insert(std::move(buffer));
 }
 
+void TrackedObjectsVK::Track(std::shared_ptr<const PipelineVK> buffer) {
```

Comment:
This looks unused?

Reply:
Oops. When pipelines with immutable samplers weren't being cached these had to be tracked in the render pass. But now the base pipeline owns that reference. Backing this out.